package.json

```diff
@@ -66,6 +66,57 @@
         "when": "confluent.chatParticipantEnabled"
       }
     ],
+    "languageModelTools": [
+      {
+        "name": "project",
+        "when": "confluent.chatParticipantEnabled",
+        "tags": [
+          "projects"
+        ],
+        "toolReferenceName": "project",
+        "displayName": "Generate Kafka Client Project",
+        "modelDescription": "Generate a project for a Kafka client. Enter parameters and autogenerate a pre-configured code starter.",
+        "canBeReferencedInPrompt": true,
+        "icon": "$(files)",
+        "inputSchema": {
+          "type": "object",
+          "properties": {
+            "cc_bootstrap_server": {
+              "type": "string",
+              "description": " One or more comma-separated host and port pairs that are the addresses where Kafka brokers accept client bootstrap requests."
```

Comment:
Might be too early to ask, but =do we know how these descriptions are interpreted? Do we need to make sure they have repeat context or have we played with description adjustments on the prompts that aren't consistently responding yet?

Reply:
Not sure yet for the input schema properties themselves, but the `modelDescription` values at the tool level will definitely take some tuning